### PR TITLE
feature/#12

### LINKED ISSUE #12: Add  Host Configuration in project
### Description
Add Host Configuration in project

### Requirements
- project must be runnable
- add middleware
- register Exposers
- install swager
- register swager

### PR DESCRIPTION
**Pull request by rashidovasadbek**
close #12

### CODE REVIEW

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    /// Configures application builder
+    /// </summary>
+    /// <param name="builder">Application builder</param>
+    /// <returns>Application builder</returns>
+    public static ValueTask<WebApplicationBuilder> ConfigureAsync(this WebApplicationBuilder builder)
+    {
+        builder
+            .AddExposers()
+            .AddDevTools();
+        return new(builder);
```
Comment (botirjonabdumannonovv) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs:15]: should be one row in each other ...

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    
+    /// <summary>
+    /// Configures application
+    /// </summary>
+    /// <param name="app">Application host</param>
+    /// <returns>Application host</returns>
+    public static  ValueTask<WebApplication> ConfigureAsync(this WebApplication app)
```
Comment (bunyodkamalov) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs:23]: Add  `async`  keyword to public static ValueTask<WebApplication> ConfigureAsync()

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        app.UseDevTools().UseExposers();
+        return new (app);
```
Comment (botirjonabdumannonovv) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs:26]: bir qator qolib ketibdi

**Review by botirjonabdumannonovv — DISMISSED** [review]

**Review by bunyodkamalov — CHANGES_REQUESTED** [review]
Change requestred changes

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs (hunk 2 above)
Comment (rashidovasadbek) [reply src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs:23]: here async keyword is written later, for example, when registering seed data

Comment (bunyodkamalov) [reply src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs:24]: Ok

**Review by bunyodkamalov — APPROVED** [review]

**Review by botirjonabdumannonovv — APPROVED** [review]

**Pull request merged**